products/azure-devops.md

```diff
@@ -13,38 +13,68 @@ activeSupportColumn: true
 releaseDateColumn: true
 
 releases:
--   releaseCycle: "2022"
+-   releaseCycle: "2022.1"
+    releaseLabel: "2022 Update 1"
+    releaseDate: 2023-11-28
+    support: 2028-01-11
+    eol: 2033-01-11
+    latest: "2022.1"
+    latestReleaseDate: 2023-11-28
+    link: https://learn.microsoft.com/azure/devops/server/release-notes/azuredevops2022u1#azure-devops-server-2022-update-1-release-date-november-28-2023
+
+-   releaseCycle: "2022.0"
+    releaseLabel: "2022"
     releaseDate: 2022-12-06
     support: 2028-01-11
     eol: 2033-01-11
-    latest: "2022.0.1patch4"
-    latestReleaseDate: 2023-10-10
-    link: https://learn.microsoft.com/azure/devops/server/release-notes/azuredevops2022?view=azure-devops-2020#azure-devops-server-2022-update-01-patch-4-release-date-october-10-2023
+    latest: "2022.0.1patch5"
+    latestReleaseDate: 2023-11-14
+    link: https://learn.microsoft.com/azure/devops/server/release-notes/azuredevops2022#azure-devops-server-2022-update-01-patch-5-release-date-november-14-2023
 
--   releaseCycle: "2020"
+-   releaseCycle: "2020.1"
+    releaseLabel: "2020 Update 1"
+    releaseDate: 2021-05-25
+    support: 2025-10-14
+    eol: 2030-10-08
+    latest: "2020.1.2patch10"
+    latestReleaseDate: 2023-11-14
+    link: https://learn.microsoft.com/azure/devops/server/release-notes/azuredevops2020u1#azure-devops-server-2020-update-12-patch-10-release-date-november-14-2023
+
+-   releaseCycle: "2020.0"
+    releaseLabel: "2020"
     releaseDate: 2020-08-25
     support: 2025-10-14
     eol: 2030-10-08
-    latest: "2020.1.2patch9"
-    latestReleaseDate: 2023-10-10
-    link: https://learn.microsoft.com/azure/devops/server/release-notes/azuredevops2020u1?view=azure-devops-2020#azure-devops-server-2020-update-12-patch-9-release-date-october-10-2023
+    latest: "2020.0.2patch6"
+    latestReleaseDate: 2023-11-14
+    link: https://learn.microsoft.com/azure/devops/server/release-notes/azuredevops2020#azure-devops-server-2020-update-02-patch-6-release-date-november-14-2023
+
+-   releaseCycle: "2019.1"
+    releaseLabel: "2019 Update 1"
+    releaseDate: 2019-08-20
+    support: 2024-04-09
+    eol: 2029-04-10
+    latest: "2019.1.2patch6"
+    latestReleaseDate: 2023-11-14
+    link: https://learn.microsoft.com/azure/devops/server/release-notes/azuredevops2019u1#azure-devops-server-2019-update-12-patch-6-release-date-november-14-2023
 
--   releaseCycle: "2019"
+-   releaseCycle: "2019.0"
+    releaseLabel: "2019"
     releaseDate: 2019-03-05
     support: 2024-04-09
     eol: 2029-04-10
-    latest: "2019.1.2patch5"
-    latestReleaseDate: 2023-09-12
-    link: https://learn.microsoft.com/azure/devops/server/release-notes/azuredevops2019u1?view=azure-devops-2020#azure-devops-server-2019-update-12-patch-5-release-date-september-12-2023
+    latest: "2019.0.2patch16"
```

Comment:
```suggestion
    latest: "2019.0.1patch16"
```

According to https://devblogs.microsoft.com/devops/november-patches-for-azure-devops-server-and-team-foundation-server/.